ui/src/app/store/machine/utils/pod.ts

```diff
@@ -0,0 +1,19 @@
+import type { Machine } from "app/store/machine/types";
+import type { Pod } from "app/store/pod/types";
+
+export const getPodNumaID = (machine: Machine, pod: Pod): number | null => {
```

Comment:
Should this one maybe live in store/pod/utils.ts? It feels like it's more about a pod attribute than a machine attribute, but maybe I'm wrong.

Reply:
Yeah I think you're right, just so happens that it's used in machine details